.github/workflows/notify_on_tag_creation.yml

```diff
@@ -0,0 +1,40 @@
+name: Send POST Request on Tag Creation
+
+on:
+  create:
+    tags:
+      - "rc*"
+env:
+  NOTION_DATABASE_ID: e3aca840563b4cc8b273e65da748e1dd
+  WEBHOOK_URL: https://hook.eu2.make.com/sg1xhhlvnp4314llgde92csqgjrpm42e
+
+jobs:
+  send_post_request:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up environment variables
+        run: |
+          echo "NOTION_DATABASE_ID=e3aca840563b4cc8b273e65da748e1dd" >> $GITHUB_ENV
+          echo "WEBHOOK_URL=https://hook.eu2.make.com/sg1xhhlvnp4314llgde92csqgjrpm42e" >> $GITHUB_ENV
```

Comment:
Can we remove these lines now because we have this at the top?
```
env:
  NOTION_DATABASE_ID: e3aca840563b4cc8b273e65da748e1dd
  WEBHOOK_URL: https://hook.eu2.make.com/sg1xhhlvnp4314llgde92csqgjrpm42e
```